pos_customer_wallet/tests/test_balance.py

```diff
@@ -68,8 +68,6 @@ def test_close_session(self):
         )
         # One line for each partner
         self.assertEqual(len(move_lines), 2)
-        # Two credits and one debit on the move
-        self.assertEqual(len(move_lines[0].move_id.line_ids), 3)
```

Comment:
could you just comment the line. So we can investigate in a second step ?

Reply:
done